web/client/actions/mapInfo.js

```diff
@@ -193,6 +195,20 @@ function featureInfoClick(point, layer) {
     };
 }
 
+function highlightFeature(enabled) {
```

Comment:
This enables / disables the highlightFeature functionality, it doesn't highlight a feature, I would change the name of the action and action creator.

Reply:
ok

Reply:
any suggestion?